CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ option(NETWORKIT_WARNINGS_AS_ERRORS "Treat warnings as errors (except deprecated
 option(NETWORKIT_CLANG_TIDY "Check code with clang-tidy" OFF)
 option(NETWORKIT_FLATINSTALL "Install into a flat directory structure (useful when building a Python package)" OFF)
 option(NETWORKIT_COVERAGE "Build with support for coverage" OFF)
+option(NETWORKIT_SANITY_CHECKS "Algorithms will perform further (expensive) checks" OFF)
```

Comment:
The flag actually needs to be exported to the compiler by setting the `-DNETWORKIT_SANITY_CHECKS` option of the compiler's command line.

Reply:
So should we use a macro for additional tests? How would that work exactly?